hash/ibm5170_cdrom.xml

```diff
@@ -5127,6 +5155,21 @@ The Curse of Monkey Island (Windows 95 demo version)
 		</part>
 	</software>
 
+	<!-- Windows 95/98/NT4 -->
+	<!-- Graphics bugged on svga_s3, sound not working properly on sblaster_16 -->
```

Comment:
Need more context on this: is it requiring a true color display hence the "bugged"? Post a picture.

Reply:
That's typical true color bug for the (hackish) `svga_s3` (or even, the underlying ibm8514/a adapter). What are the minimum requirements, I expect this to be better suited for `pcipc` considering the '99 release?

Reply:
The CD doesn't state any minimums beyond Windows 95 and NT 4.0, but installing on ct486 is extremely slow. I have not tried to do it in pcipc :)

Reply:
Well I'll eventually look at this, just please be more specific with the comment with the `True Color' tag (not asking to translate to a `notes` field because that needs upgrading for `ibm5170_cdrom.xml` anyway).

Reply:
Been trying pcipc, but I think the driver is nearly completely broken. Couldn't get Win95 nor NT4 to install; got Windows 2000 installed but even the basic OS will frequently crash the driver. Can't even try to get to the point of installing extra software.

I can append the comment, I didn't know there were known true color bugs; Windows 95 wasn't even set to True Color, but I used svga_s3 because it was the only one Win95 really has a driver for.

Reply:
`pcipc` BIOS is pretty finnicky, it's definitely possible to install Windows 98 on it (and the Jp DOS/V variant) but first off you have to replace PS/2 mouse with a serial one (including disabling former in CMOS). Legit never tried 95/NT4, I know 2000 has issues with the [virge card](https://github.com/mamedev/mame/blob/51a8ec962bef3a44fa6686faeb851722973e2014/src/devices/video/s3virge.cpp#L10) it uses.

Reply:
I could try 98, I didn't figure it was worth trying if 95 didn't work. Windows 2000 is fine with real ViRGE cards, so that is almost surely a MAME bug itself :p

Reply:
Well, tried twice, no good. Both Windows 98 and Me are locking up in the 16-bit GUI installer just as Win95 does, just as Windows 2000 does with certain events (opening Solitaire or the command prompt are among them).